tests/cases/text/cursor_move.slint

```diff
@@ -55,8 +62,8 @@ slint::testing::send_keyboard_string_sequence(&instance, &LEFT_CODE.to_string())
 slint::testing::send_keyboard_string_sequence(&instance, &LEFT_CODE.to_string());
 assert_eq!(instance.get_test_cursor_pos(), 0);
 
-slint::testing::set_current_keyboard_modifiers(&instance, shift_modifier);
-slint::testing::send_keyboard_string_sequence(&instance, &END_CODE.to_string());
+slint::testing::set_current_keyboard_modifiers(&instance, move_mod_shift_modifier);
```

Comment:
I'm just curious: What does "mod" in "move_mod_shift_modifier" stand for?

Reply:
"move modifier and shift". "move modifier" is either alt or control depending on the platform.
yes, modifier is in there twice. "move and shift" doesn't have the same meaning though.

Reply:
Thanks :)